### ISSUE TITLE
Question about the Configuration of Training

### ISSUE THREAD

**Issue by Quantuman134** [zheng95z/rgbx#18]
Amazing work!
Hi, I recently try to fine tune the x2rgb model with a small-batch dataset (about 1000 pair of data). I found there is no description about the training configuration in the paper. So, could you please provide some information regarding it? like type of optimizer, learning rate, etc. Thanks a lot!
